integration-cli/docker_cli_rmi_test.go

```diff
@@ -14,26 +15,22 @@ func (s *DockerSuite) TestRmiWithContainerFails(c *check.C) {
 
 	// create a container
 	out, _, err := dockerCmdWithError("run", "-d", "busybox", "true")
-	if err != nil {
-		c.Fatalf("failed to create a container: %s, %v", out, err)
-	}
+	// failed to create a container
+	c.Assert(err, checker.IsNil, check.Commentf("out: %s", out))
```

Comment:
Do you mean that  use
```
out, _ := dockerCmd("run", "-d", "busybox", "true")
```
to replace
```
out, _, err := dockerCmdWithError("run", "-d", "busybox", "true")
c.Assert(err, checker.IsNil, check.Commentf("out: %s", out))
```
If so, do you really think it's OK to ignore failure of `docker run`? I really doubt it...